crates/evm/abi/src/console/ds.rs

```diff
@@ -0,0 +1,83 @@
+//! DSTest log interface.
+
+use super::{format_units_int, format_units_uint};
+use alloy_primitives::hex;
+use alloy_sol_types::sol;
+use derive_more::Display;
+use itertools::Itertools;
+
+// TODO: Use `UiFmt`
```

Comment:
Is this still desired, if so - why?

Reply:
to share the same formatting implementation